agent-codeless/package/Microsoft.ApplicationInsights.JavaCodelessAgent.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata>
     <id>Microsoft.ApplicationInsights.JavaCodelessAgent</id>
-    <version>2.5.1</version>
+    <version>2.5.2</version>
```

Comment:
@trask 2.5.2 or 2.6.0?

Reply:
somewhere in between? 😄

i'm good with `2.5.2`